clang/test/Parser/cxx2c-oxford-variadic-comma.cpp

```diff
@@ -0,0 +1,36 @@
+// RUN: %clang_cc1 -std=c++2c -fsyntax-only -verify %s
+
+void a(...);
```

Comment:
A few more test cases that came to mind:
```c++
struct S {
    void g(this S...) {}
};

template <typename ...Ts>
void f(Ts......) {}

template <typename T>
void g(T...) {}

```

Reply:
Thanks! I've added these additional test cases.